cypress/utils/users.js

```diff
@@ -0,0 +1,40 @@
+import { getPermissionsArray } from "../apiRequests/permissions";
+import { inviteStaffMember } from "../apiRequests/staffMember";
+
+export function inviteStaffMemberWithFirstPermission({
+  email,
+  isActive = true,
+  firstName = "",
+  lastName = ""
+}) {
+  return getPermissionsArray(1).then(permissions => {
+    inviteStaffMember({
+      firstName,
+      lastName,
+      email,
+      isActive,
+      permissionId: permissions[0].node.id
+    });
+  });
+}
+
+export function getMailActivationLinkForUser(email, i = 0) {
```

Comment:
This function is very cryptic, why is it recursive? If it's supposed to check few times before failing there should be some timeout, otherwise it's synchronously calling `cy.mhGetMailsByRecipient` few times immediately.

Reply:
There is timeout. cy.mhGetMailsByRecipient(email).should - should by default has timeout

Reply:
I agree with @jwm0, but if this function needs to be recursive, then can we add a comment to explain this is purposeful?

Reply:
Comment like this will be transparent? "Function mhGetMailsByRecipient first get all emails from mailhog with a timout, and after that it finds email from recipient. It cloud happened that invite email from saleor will not be recived yet, so in this case I want to take again all of the emails and check if there is an email to the correct recipient"